docs/migration/migration1120to1130.adoc

```diff
@@ -120,3 +120,14 @@ A változtatások nem eredményeznek átállási munkálatokat, visszafelé komp
 ==== Átállás
 
 A változtatások nem eredményeznek átállási munkálatokat, visszafelé kompatibilis.
+
+=== coffee-rest
+
+* Bevezetésre került az `OptimisticLockException`, alapértelmezetten `CoffeeFaultType.OPTIMISTIC_LOCK_EXCEPTION` hibakóddal.
+* Átalakításra került a `DefaultBaseExceptionMapper` osztályban a hibakezelés:
+* Uzleti hiba - BusinessException esetén 422 státuszkódot adunk az eddigi 500 Státuszkód helyett.
+* Technikai hiba - OptimisticLockException esetén 518 státuszkódot adunk az eddigi 500 Státuszkód helyett.
+
+==== Átállás
+
+* Az visszafelé kompatibilításhoz a projekteken saját ExceptionMapper létrehozása szükséges.
```

Comment:
Visszafelé nem kompatibilis, hiszen más státuszkódot ad. Erre figyelmeztetni kell a klienseket. Ezen kívül egy felhívást is lehet bele kéne írni, miszerint a BaseException helyett mindenhol megfelelő Business vagy Technical exception-t kéne használni, hogy megfelelő legyen a státuszkód.

Reply:
Ezért írtam, hogy szükséges hozzá saját ExceptionMapper de akkor félre érthető a megfogalmazás átírom.  A felhívás-t is ugyanide írjam?